From a4e6d59a6c0f841fb36c59294ef4c68688bf011b Mon Sep 17 00:00:00 2001
From: vietn <vietn@fsoft.com.vn>
Date: Sat, 26 May 2018 15:14:31 +0700
Subject: [PATCH 590/628] Revert "drm: rcar-du: Fix hot plug detection method"

This reverts commit f604460a4ceb ("drm: rcar-du: Fix hot
	plug detection method")

The commit make an issue of display on iwg20m-g1n platform,
more testing is needed, revert it for now

Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNv25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 drivers/gpu/drm/rcar-du/rcar_du_hdmicon.c | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_hdmicon.c b/drivers/gpu/drm/rcar-du/rcar_du_hdmicon.c
index 0d36e82..6c92714 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_hdmicon.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_hdmicon.c
@@ -16,8 +16,6 @@
 #include <drm/drm_crtc.h>
 #include <drm/drm_crtc_helper.h>
 #include <drm/drm_encoder_slave.h>
-#include <linux/gpio.h>
-#include <linux/of_gpio.h>
 
 #include "rcar_du_drv.h"
 #include "rcar_du_encoder.h"
@@ -87,7 +85,6 @@ int rcar_du_hdmi_connector_init(struct rcar_du_device *rcdu,
 	struct rcar_du_connector *rcon;
 	struct drm_connector *connector;
 	int ret;
-	struct device_node *np;
 
 	rcon = devm_kzalloc(rcdu->dev, sizeof(*rcon), GFP_KERNEL);
 	if (rcon == NULL)
@@ -97,12 +94,7 @@ int rcar_du_hdmi_connector_init(struct rcar_du_device *rcdu,
 	connector->display_info.width_mm = 0;
 	connector->display_info.height_mm = 0;
 	connector->interlace_allowed = true;
-	np = of_find_node_by_name(NULL, "adv7511w");
-	if ((np) && (of_get_gpio(np, 0) > 0))
-		connector->polled = DRM_CONNECTOR_POLL_HPD;
-	else
-		connector->polled = DRM_CONNECTOR_POLL_CONNECT |
-							DRM_CONNECTOR_POLL_DISCONNECT;
+	connector->polled = DRM_CONNECTOR_POLL_HPD;
 
 	ret = drm_connector_init(rcdu->ddev, connector, &connector_funcs,
 				 DRM_MODE_CONNECTOR_HDMIA);
-- 
2.7.4

